src/components/SortButton/SortButton.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import './SortButton.css'
+import less from '../../assets/icons/expand_less.svg'
+import more from '../../assets/icons/expand_more.svg'
+
+const style = {
+  styleSortBtn: 'SortButtonClass'
+}
+
+interface ButtonProps {
+  /**
+   *  Texto do botão
+   */
+  text: string,
+  /**
+   * Informa o estado do botal
```

Comment:
Só um errinho de digitação na documentação
```suggestion
   * Informa o estado do botão
```

Reply:
Corrigido @joellacerda